src/deployment/deployment-hapi-plugin.ts

```diff
@@ -146,20 +144,19 @@ class DeploymentHapiPlugin {
     }, {
       method: 'GET',
       path: '/raw-deployment-handler/{param*}',
-      handler: this.directoryHandler('key'),
+      handler: this.directoryHandler(PREKEY),
       config: {
         auth: 'customAuthorize',
```

Comment:
The point of `customAuthorize` (an authentication strategy defined in `AuthenticationHapiPlugin`) is to only authenticate at the top level and leave the authorization for later.